Storage/SQL/SQLiteHistory.swift

```diff
@@ -107,15 +107,15 @@ public class SQLiteHistory {
     let favicons: FaviconsTable<Favicon>
     let prefs: Prefs
 
-    required public init?(db: BrowserDB, prefs: Prefs) {
+    required public init(db: BrowserDB, prefs: Prefs) {
         self.db = db
         self.favicons = FaviconsTable<Favicon>()
         self.prefs = prefs
 
         // BrowserTable exists only to perform create/update etc. operations -- it's not
         // a queryable thing that needs to stick around.
         if !db.createOrUpdate(BrowserTable()) {
-            return nil
+            log.error("Unable to create or update browser table!")
```

Comment:
The reason I think we should crash: if the DB is open, but we fail to set up our schema, we're screwed, and continuing will cause a rash of weird errors throughout the app. Better to die and catch it in our crash reporter.